### PR TITLE
Make initial patches for arm support

### PR DESCRIPTION
**Pull request by gurnben**
## Summary of Changes

This is an initial set of changes to support the image-builder on ARM.

### CODE REVIEW

File: build/setup-oc.sh [hunk 1]
```diff
@@ -8,6 +8,8 @@ if [ "$(uname -m)" = "s390x" ]; then
         url="http://mirror.openshift.com/pub/openshift-v4/s390x/clients/ocp/latest/openshift-client-linux.tar.gz"
 elif [ "$(uname -m)" = "x86_64" ];  then
         url="https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/openshift-client-linux.tar.gz"
+elif [ "$(uname -m)" = "aarch64" ];  then
+        url="https://mirror.openshift.com/pub/openshift-v4/aarch64/clients/ocp/latest/openshift-client-linux.tar.gz"
```
Comment (deepsm007) [inline build/setup-oc.sh:12]: Can we make this more generic? Something like 
` url="https://mirror.openshift.com/pub/openshift-v4/$(uname -m)/clients/ocp/latest/openshift-client-linux.tar.gz"`

**Review by deepsm007 — COMMENTED** [review]
Can we make

File: build/setup-oc.sh (hunk 1 above)
Comment (gurnben) [reply build/setup-oc.sh:12]: Good point - I'll admit - I was concerned by the odd specificity of the other lines, so I mirrored them - let's go ahead and make it the generic line that should cover all cases

**Pull request merged**